server/tests/location.test.ts

```diff
@@ -0,0 +1,118 @@
+import request from 'supertest';
+import { expect } from 'chai';
+import app from '../src/app';
+import authorize from './utils/auth';
+import { Location } from '../src/models';
+import { clearDB, populateDB } from './utils/db';
+import { AdminType } from '../src/models/admin';
+import { LocationType, Tag } from '../src/models/location';
+import { DriverType } from '../src/models/driver';
+import { RiderType } from '../src/models/rider';
+
+const testAdmin: Omit<AdminType, 'id'> = {
+  firstName: 'Test-Admin',
+  lastName: 'Test-Admin',
+  phoneNumber: '1111111111',
+  email: 'test-admin@cornell.edu',
+};
+
+const testDriver: Omit<DriverType, 'id'> = {
+  firstName: 'Test-Driver',
+  lastName: 'Test-Driver',
+  availability: {
+    Mon: undefined,
+    Tue: undefined,
+    Wed: undefined,
+    Thu: undefined,
+    Fri: undefined,
+  },
+  vehicle: {
```

Comment:
it should be a `vehicle id` instead of the vehicle object. we are using `populate()` to populate this field. DriverType has `vehicle:VehicleType` but it should be string (the `id`).

Reply:
Do we have types for what request bodies?

Reply:
My mistake, this isn't about request bodies. I mean for what the db structure looks like.

Reply:
All the types are in the `src/models`. There are currently no object in the database that actually has the vehicle populated, so we were not getting any errors. However, the current implementation in the types is incorrect. Vehicle field is marked as `VehicleType` but it should be simply `string` because we put the `id` not the entire object.

I asked harrison to fix that in his Driver endpoint test pr.

Unfortunately, we can't use something lie an ObjectId in dynamoose to specify that the `vehicle` is an ObjectId (mongoose has that, but not dynamoose). So, we need to stick with string. I will soon work on fixing all these types issues + updating notion for API